lib/jasmine/add-custom-matchers.6.js

```diff
@@ -6,8 +6,12 @@
 
 /* global beforeEach, jasmine */
 
-const customMatchers = require('./custom-matchers');
+import {toDeepEqual, toBeStubComponent, toHaveCorrectPropTypes} from './custom-matchers';
 
 beforeEach(() => {
-    jasmine.addMatchers(customMatchers);
+    jasmine.addMatchers({
+        toDeepEqual,
+        toBeStubComponent,
+        toHaveCorrectPropTypes,
+    });
```

Comment:
With this refactor adding custom matchers now means adding the matcher to `custom-matchers` as well as adding an import to this file. Was this extra step intended or should we leave it the old way so you only have to add a custom matcher to `custom-matchers`?